components.yaml

```diff
@@ -41,9 +41,3 @@ components:
     branch: main
     update-policy: tagged
     metadata: v0.29.1
-  kube-secondary-dns:
```

Comment:
I don't think removing this component entirely is a good idea, we still want to be able to manually bump, and for that we need this.

Can I suggest that we change the bumper so we don't exitWithError if it fails handling a specific component?

Reply:
yeah that's what I meant

Regarding my suggestion, if we allow a component to fail auto-bump and continue to the other components, then at least we won't clock other components..